data/json/scenarios.json

```diff
@@ -491,5 +491,16 @@
     "start_name": "Shady Basement",
     "allowed_locs": [ "basement_bionic" ],
     "professions": [ "cyberjunkie", "faulty_bionic", "cykotic", "razorgirl", "bionic_installer" ]
+  },
+  {
+    "type": "scenario",
+    "ident": "overrun",
+    "name": "Overrun",
+    "points": 4,
+    "description": "When your base got overrun by the dead, your last order was to retreat to the armory.  During all the chaos you got separated from your squad and you are now stuck in the warehose all alone.  You are not sure if anyone made it to the armory, or you are the last man alive.",
+    "start_name": "Military Base Warehouse",
+    "allowed_locs": [ "mil_base_2g" ],
+    "professions": [ "soldier" ],
```

Comment:
Not at least allowing the same military professions as allowed in Helicopter crash?

Reply:
Added specops, bio_soldier and bio_sniper to starting professions.